uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_009
# Pull request: Production websokect issues:

## What does this PR do?  
This PR resolves issues related to Action Cable and WebSockets in the production environment. The changes ensure stable real-time communication functionality and improve connection reliability.

## Testing  
1. Deploy the updated code to the staging environment.  
2. Test real-time features like notifications and live updates using Action Cable in staging.  
3. Verify WebSocket connections remain stable under typical user loads.  
4. Confirm no errors in the logs related to Action Cable or WebSockets.  

## Checklist  

## Extra information (if needed)  
Performance logs indicate improved stability with no observed WebSocket disconnects under stress tests.
